### PR TITLE
Add recipe for fault-localization

### PR DESCRIPTION
**Pull request by nicoddemus**
https://github.com/hchasestevens/fault-localization

### CODE REVIEW

**Comment by nicoddemus** [comment]
@conda-forge/staged-recipes @ocefpaf I believe the package is ready for review/merge. 👍

File: recipes/fault-localization/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{% set name = "fault-localization" %}
+{% set version = "0.1.3" %}
+{% set file_ext = "tar.gz" %}
```
Comment (ocefpaf) [inline recipes/fault-localization/meta.yaml]: We need to fix skeleton and the example to avoid this excess of jinja vars :-(

Comment (nicoddemus) [reply recipes/fault-localization/meta.yaml]: Yeah the `file_ext` seems a bit too much, specially that PyPI is planning to standardize on `tar.gz` only.

**Comment by nicoddemus** [comment]
Thanks @ocefpaf!

**Comment by ocefpaf** [comment]
We are having trouble with Travis-CI and many feedstocks were not created.
I know that @isuruf already identified one possible reason for this and he is working on it. (I'm tied up with classes this week but I'll try to investigate this later today when I get a break.)

**Comment by nicoddemus** [comment]
OK, thanks @ocefpaf!

**Pull request merged**